packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -242,6 +244,10 @@ export class CdkToolkit {
         error('\n ❌  %s failed: %s', colors.bold(stack.displayName), e);
         throw e;
       } finally {
+        if (options.cloudWatchLogMonitor) {
+          const result = await registerCloudWatchLogGroups(this.props.sdkProvider, stack);
```

Comment:
`result` seems way too generic of a name.

`foundLogGroupsResult`, or something?